models/listing.js

```diff
@@ -117,12 +117,8 @@ exports.createListingForUserID = function(userid, listing, callback = { success:
 		INSERT INTO Listing (LS_US_Seller, LS_Title, LS_Description, LS_Price, LS_RemainingStock, LS_IsActive)
 		VALUES (?, ?, ?, ?, ?, 1);
 	`
-	if (listing.remainingStock == null)
```

Comment:
You should actually keep this here, this makes it that if there is no remaining stock value set, it will default to 1.